src/components/Output/Output.js

```diff
@@ -137,20 +157,27 @@ class Output extends React.Component {
     </div>
   );
 
-  runCode = () => {
-    this.setState((prevState) => ({
-      run: prevState.run + 1,
-    }));
-  };
-
   render() {
     return (
       <div className="editor-output">
-        {this.renderBanner()}
-        <div>{this.renderOutput()}</div>
+          { this.renderBanner() }
+          <div
+            id={`${this.state.counter} ${this.state.run}`}
+            key={`${this.state.counter} ${this.state.run}`}
+            className="editor-output-iframe"
+            style={{ height: `${this.props.screenHeight - 61}px` }}
+          >
+            <div id="output">
+              <textarea id="inner" readOnly></textarea>
+            </div>
+            <canvas id="my-canvas"></canvas>
```

Comment:
Hey @8BitRobot , you kind of got baited by the name, which is completely understandable. The canvas should be a `div`, not a `canvas`. Hopefully that should make everything easier for you :)